### DISCUSSION TITLE
post/ios-manage-resources-in-multi-framework/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#78] [Announcements]
# post/ios-manage-resources-in-multi-framework/

앱 용량에 큰 영향을 끼치는 요소인 리소스를 효율적으로 관리하기 위해 리소스 모듈을 만든 과정을 소개합니다.

https://tech.kakaopay.com/post/ios-manage-resources-in-multi-framework/

**Comment by pay-dan-dy** [comment]
리소스를 효율적으로 사용하기 위한 고민과 해결 공유 감사합니다~
